server/src/lib/model/db.ts

```diff
@@ -307,7 +314,7 @@ export default class DB {
       taxonomySentences = await this.findSentencesMatchingTaxonomy(
         client_id,
         locale_id,
-        count,
+        count + 5, // 5 might be removed later
```

Comment:
Probably better to add a constant here we a meaningful name
```suggestion
        count + CLIPS_BUFFER,
```